src/bootstrap/config.rs

```diff
@@ -200,6 +200,9 @@ pub struct Config {
     pub initial_rustc: PathBuf,
     pub initial_rustfmt: Option<PathBuf>,
     pub out: PathBuf,
+
+    /// Clippy **A**llow **D**eny **W**arn **F**orbid
+    pub clippy_adwf: [Option<String>; 4],
```

Comment:
this should be named clippy_lint_level or something like that I think

Reply:
Renamed in [`66e4bb9` (#97210)](https://github.com/rust-lang/rust/pull/97210/commits/66e4bb9b18a55c485b1ef11832836cf64321c40c)